<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
</head>
<body>
<h1 id="education">Education</h1>
<p><span> <span><strong>Master of Science, Bioinformatics</strong></span> Sept.2014 – May.2017<br />
</span></p>
<ul>
<li><p>University of British Columbia, Vancouver, BC</p></li>
<li><p>Funded by CIHR and the CGS-M (see Awards section)</p></li>
<li><p>Thesis project (see Projects section)</p></li>
</ul>
<p><span> <span><strong>Bachelor of Science, Biochemistry</strong></span> Sept.2008 – May.2013<br />
</span></p>
<ul>
<li><p>University of British Columbia, Vancouver, BC</p></li>
<li><p>Completed the Co-operative Education Program</p></li>
</ul>
<h1 id="awards">Awards</h1>
<p>2016 <span><strong>CEEHRC Travel Award recipient</strong></span><br />
 </p>
<p><span id="tab"></span>&#8627;&nbsp;Awarded $1,000 for the cost of travel to present at the ISMB 2016 conference in Orlando, Florida.</p>
<p>2016 <span><strong>John Bosdet Memorial Fund</strong></span><br />
 </p>
<p>2015 <span><strong>Canada Graduate Scholarship Master’s Award (CGS-M) recipient</strong></span><br />
 </p>
<p><span id="tab"></span>&#8627;&nbsp;Administered by the Tri-Council Agencies (NSERC, SSHRC, CIHR), the CGS-M is a one year, non-renewable research training award valued at $17,500 to support students who demonstrate a high standard of achievement in undergraduate and early graduate studies.</p>
<p>2014 <span><strong>CIHR Strategic Training in Health Research Award recipient</strong></span><br />
 </p>
<p><span id="tab"></span>&#8627;&nbsp;Awarded a two-year ($22,000/year) funding for M.Sc. training in Bioinformatics.</p>
<p>2013 <span><strong>NSERC Undergraduate Student Research Award</strong></span><br />
 </p>
<p>2008 <span><strong>UBC President’s Entrance Scholarship</strong></span><br />
 </p>
<p><span id="tab"></span>&#8627;&nbsp;This one-time funding of $2,000 is to support academically qualified students who show an interest in joining the UBC community.</p>
<h1 id="publications">Publications</h1>
<p>Co-authored in:</p>
<p>Siu et al. (2017) Characterization of the human thyroid epigenome. <span><em>Journal of Endocrinology, 235</em></span>(2):153-165. doi:10.1530/JOE-17-0145</p>
<p>Lever et al. (2017) A collaborative filtering-based approach to biomedical knowledge discovery. <span><em>Bioinformatics, 34</em></span>(4):652-659. doi:10.1093/bioinformatics/btx613</p>
<p>Wang et al. (2017) Genomic consequences of aberrant DNA repair mechanisms stratify ovarian cancer histotypes. <span><em>Nature genetics, 49</em></span>(49):856-865. doi:10.1038/ng.3849</p>
<p>Wong et al. (2017) Management of PET diagnosed thyroid incidentalomas in British Columbia Canada: Critical importance of the PET report. <span><em>The American Journal of Surgery, 213</em></span>(5):950-957. doi:10.1016/j.amjsurg.2017.03.015</p>
<p>McPherson et al. (2016) Divergent modes of clonal spread and intraperitoneal mixing in high-grade serous ovarian cancer. <span><em>Nature Genetics</em></span>. doi:10.1038/ng.3573</p>
<p>Eirew et al. (2015) Dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution. <span><em>Nature, 518</em></span>(7539):422-6. doi: 10.1038/nature13952</p>
<p>Leon et al. (2012) Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text. <span><em>Bioinformatics, 28</em></span>(22):2963-70. doi: 10.1093/bioinformatics/bts542</p>
<p>Acknowledged in:</p>
<p>Zoubarev et al. (2012) Gemma: a resource for the reuse, sharing and meta-analysis of expression profiling data. <span><em>Bioinformatics, 28</em></span>(17):2272-3. doi: 10.1093/bioinformatics/bts430</p>
<h1 id="presentations">Presentations</h1>
<p><span> Poster presentation, Intelligent Systems for Molecular Biology (ISMB) 2016 Conference July 9 &amp; 10, 2016<br />
</span> <span> Poster presentation, BTP/IOP/GSAT 2016 Retreat March 11, 2016<br />
</span> <span> Oral presentation, Bioinformatics Research Rotation Presentation July 24, 2015<br />
</span> <span> Oral presentation, Bioinformatics Research Rotation Presentation March 27, 2015<br />
</span> <span> Oral presentation, Bioinformatics Research Rotation Presentation November 28, 2014<br />
</span></p>
<h1 id="projects">Projects</h1>
<p><span> <span><strong>100 Days of Code</strong></span> Feb.2017 – Jun.2017<br />
</span> <span> <span><em>See my technical blog: http://csiu.github.io/blog/tag/100daysofcode</em></span></span></p>
<ul>
<li><p>Formulated models using machine learning methods from Scikit-learn to enter Kaggle competitions</p></li>
<li><p>Setup and used PostgreSQL databases to gain experience with SQL and learn new technologies</p></li>
<li><p>Analyzed data using Python (eg. Pandas, Matplotlib, BeautifulSoup, Nltk, Theano), R, D3, &amp; APIs</p></li>
</ul>
<p><span> <span><strong>Characterization of the human thyroid epigenome</strong></span> Sept.2015 – Present<br />
</span> <span> <span><em>Jones Lab, Michael Smith Genome Sciences Centre</em></span></span></p>
<ul>
<li><p>Modelled chromatin states from epigenetic features using hidden Markov models to produce 26 candidate models</p></li>
<li><p>Developed a new model selection criterion</p></li>
<li><p>Used Makefiles to create pipelines, organize code, and rerun analyses</p></li>
<li><p>Master’s Thesis:</p></li>
</ul>
<blockquote>
<p>Siu, C. (2017). Characterization of the human thyroid epigenome (T). University of British Columbia. Retrieved from https://open.library.ubc.ca/collections/ubctheses/24/items/1.0343360</p>
</blockquote>
<p><span> <span><strong>Literature-based knowledge discovery of biomedical text</strong></span> May.2015 – Aug.2015<br />
</span> <span> <span><em>Jones Lab, Michael Smith Genome Sciences Centre</em></span></span></p>
<ul>
<li><p>Contributed to the development of text mining and machine learning approaches to prioritize therapeutics from Medline abstracts and the full-text of open access papers</p></li>
<li><p>Visualized the output of MySQL database queries in R and D3</p></li>
</ul>
<p><span> <span><strong>miRNA promoter recognition with CAGE and sRNA-seq</strong></span> Jan.2015 – Apr.2015<br />
</span> <span> <span><em>Wasserman Lab, Centre for Molecular Medicine and Therapeutics</em></span></span></p>
<ul>
<li><p>Modified a miRNA promoter recognition method to include a new feature in Python</p></li>
<li><p>Evaluated a semi supervised statistical model trained on CAGE data, DNA sequence features, and sRNA-sequencing data</p></li>
</ul>
<p><span> <span><strong>Primer Data Extraction &amp; Blast API</strong></span> Sept.2014 – Dec.2014<br />
</span> <span> <span><em>Daley Lab, Centre for Heart Lung Innovation</em></span></span></p>
<ul>
<li><p>Used data mining to extract primers sequences from academic papers</p></li>
<li><p>Made API calls to programmatically submit BLAST searches of primer sequences</p></li>
</ul>
<h1 id="employment-history">Employment history</h1>
<p><span> <span><strong>Visier Inc.</strong></span> May.2017 – Present<br />
</span> <span> <span><em>Data Management Engineer II, Lead</em></span></span><span> (Jul.2018 – Present)</span><br />
<span> <span><em>Data Management Engineer I</em></span></span><span> (May.2017 – Jul.2018)</span></p>
<ul>
<li><p>Review new customer onboarding activities and business logic customizations by checking requirements, doing design reviews, and evaluating code reviews</p></li>
<li><p>Manage the team workload in JIRA by organizing and assigning tasks to the appropriate sprints and assignee</p></li>
<li><p>Coach, mentor, and train up junior and new staff</p></li>
<li><p>Give presentations to to small and large groups during training sessions, workshops, DM all hands meetings, and new hire info session of the Data Management (DM) Team</p></li>
<li><p>Extract, transform, and load large amounts of customer data into the Visier application using Mercurial/Git and proprietary DSL</p></li>
</ul>
<p><span> <span><strong>BCCRC <span class="math inline">|</span> The University of British Columbia</strong></span> Jan.2014 – Aug.2014<br />
</span> <span> <span><em>Research Assistant at the Shah Lab</em></span></span></p>
<ul>
<li><p>Wrote Bash, Python, and R scripts for downstream analysis and pipeline development</p></li>
<li><p>Produced figures and limited text suitable for scientific manuscripts</p></li>
<li><p>Optimized analyses by leveraging parallel computing on compute clusters</p></li>
</ul>
<p><span> <span><strong>BCCRC <span class="math inline">|</span> Provincial Health Services Authority</strong></span> Sept.2013 – Dec.2013<br />
</span> <span> <span><em>Research Intern at the Shah Lab</em></span></span></p>
<ul>
<li><p>Ran automated analysis pipelines for data processing and data generation</p></li>
<li><p>Wrote scripts to rename files, parse data, and wrap third party software</p></li>
<li><p>Documented protocols, results, and comments on platforms such as the Atlassian Confluence wiki and Jira issue tracker</p></li>
</ul>
<p><span> <span><strong>NSERC Undergraduate Student Research Award</strong></span> May.2013 – Aug.2013<br />
</span> <span> <span><em>Undergraduate Summer Research Student in the UBC Department of Statistics</em></span></span></p>
<ul>
<li><p>Contributed to the collaborative development of an R package using Git and Github</p></li>
<li><p>Validated code integrity by writing and running unit tests</p></li>
</ul>
<p><span> <span><strong>UBC Centre for High-throughput Biology</strong></span> May.2012 – Aug.2012<br />
</span> <span> <span><em>System Administrator Assistant</em></span></span></p>
<ul>
<li><p>Produced and updated protocols for internal use and for end users</p></li>
<li><p>Setup user accounts on desktops, servers, and wikis</p></li>
<li><p>Monitored, backed up, and performed various system administrative tasks using Nagios, Bacula, CFengine, and Request Tracker</p></li>
</ul>
<p><span> <span><strong>UBC Centre for High-throughput Biology</strong></span> Sept.2011 – Apr.2012<br />
</span> <span> <span><em>Bioinformatics Research Assistant at the Pavlidis Lab</em></span></span></p>
<ul>
<li><p>Annotated Gene Expression Omnibus datasets that were to be imported with ontology terms</p></li>
<li><p>Evaluated software across multiple operating systems and reported issues on BugZilla</p></li>
<li><p>Assessed the quality of microarray data for the FishManOmics project headed by the Department of Fisheries and Oceans Canada</p></li>
</ul>
<p><span> <span><strong>Biofine International Inc.</strong></span> May.2011 – Aug.2011<br />
</span> <span> <span><em>Organic Chemistry Synthesis Laboratory Assistant</em></span></span></p>
<ul>
<li><p>Verbally communicated with supervisors to receive instructions for the execution of experiments</p></li>
<li><p>Researched, experimented, and evaluated the method of Marfey’s reaction for enantiomeric resolution of amino acids</p></li>
<li><p>Multitasked in coordination between executing experiments, keeping records, restocking supplies, and cleaning glassware</p></li>
</ul>
<p><span> <span><strong>A&amp;W Food Services of Canada</strong></span> July.2007 – Oct.2010<br />
</span> <span> <span><em>Cashier</em></span></span></p>
<ul>
<li><p>Provided excellent customer service by responding in a timely manner, listening to what the customer has to say, and keeping an upbeat positive attitude</p></li>
<li><p>Took initiative to find extra tasks when scheduled duties were completed</p></li>
</ul>
<p><span> <span><strong>Pacific National Exhibition</strong></span> Fair time<br />
</span> <span> <span><em>Concessions Attendant at Triple-O’s in August 2006, 2007, 2008, and 2009</em></span></span></p>
<ul>
<li><p>Maintained high standards of customer service during high-volume, fast-paced operations</p></li>
<li><p>Handled currency and credit transactions quickly and accurately</p></li>
</ul>
<h1 id="volunteer-experience">Volunteer experience</h1>
<p><span> <span><em>Conference Volunteer</em></span>, <span><strong>VR/AR Global Summit</strong></span> Sept.2018</span></p>
<ul>
<li><p>Greeted guests/speakers/exhibitors, provided directions, and made sure things go according to plan</p></li>
</ul>
<p><span> <span><em>Judge</em></span>, <span><strong>Technovation Challenge</strong></span> May.2018</span></p>
<ul>
<li><p>Inspired the next generation of innovators by motivating girls ages 10-18 to learn and improve their skills by providing constructive feedback to continue to develop their product</p></li>
</ul>
<p><span> <span><em>Helper</em></span>, <span><strong>Software Carpentry workshop</strong></span> Oct.2016 &amp; Feb.2017</span></p>
<ul>
<li><p>Troubleshot and helped students learn Git, GitHub, Bash, Python, and Jupyter Notebook</p></li>
</ul>
<p><span> <span><em>Wayfinder &amp; Healthy Heart Resource Centre Clerk</em></span>, <span><strong>St. Paul’s Hospital</strong></span> Feb.2013 – Aug.2013</span></p>
<ul>
<li><p>Approached, greeted, and helped patients and visitors find their way</p></li>
<li><p>Welcomed and provided information and assistance to visitors of the resource centre</p></li>
</ul>
</body>
</html>
