<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="css/main.css" type="text/css" />
</head>
<body>
<h1 id="education">Education</h1>
<p><span> <span><strong>Pursuing a Master’s degree, Bioinformatics</strong></span> Sept.2014 - Present<br />
</span></p>
<ul>
<li><p>University of British Columbia, Vancouver, BC</p></li>
<li><p>Supervised by Dr. Steven Jones at the Genome Sciences Centre, BC Cancer Agency</p></li>
<li><p>4 month lab rotations with Steven Jones, Wyeth Wasserman, and Denise Daley</p></li>
</ul>
<p><span> <span><strong>Bachelor of Science, Biochemistry</strong></span> Sept.2008 – May.2013<br />
</span></p>
<ul>
<li><p>University of British Columbia, Vancouver, BC</p></li>
<li><p>Completed the Co-operative Education Program</p></li>
</ul>
<h1 id="awards">Awards</h1>
<table>
<tbody>
<tr class="odd">
<td align="left"><span> 2016 CEEHRC Travel Award<br />
</span> <span> 2016 John Bosdet Memorial Fund<br />
</span> <span> 2015 NSERC Canada Graduate Scholarship Master’s Award<br />
</span> <span> 2014 CIHR Strategic Training in Health Research in Bioinformatics Award<br />
</span> <span> 2013 NSERC Undergraduate Student Research Award (USRA)<br />
</span> <span> 2008 UBC President’s Entrance Scholarship<br />
</span></td>
</tr>
</tbody>
</table>
<h1 id="thesis">Thesis</h1>
<p>``Characterization of the normal thyroid reference epigenome&quot;<br />
In this study, we compare the consistency of chromatin state annotations across the epigenomes from the grossly uninvolved tumour-adjacent thyroid tissue of four human individuals using ChIP-sequencing and RNA-sequencing data. We examine 6 histone modifications, identify chromatin states using hidden Markov models, produce a new metric for model selection, and establish epigenomic maps of 19 chromatin states.</p>
<h1 id="publications">Publications</h1>
<p>Co-authored in:</p>
<p>McPherson et al. (2016) Divergent modes of clonal spread and intraperitoneal mixing in high-grade serous ovarian cancer. <span><em>Nature Genetics</em></span>. doi:10.1038/ng.3573</p>
<p>Eirew et al. (2015) Dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution. <span><em>Nature, 518</em></span>(7539):422-6. doi: 10.1038/nature13952</p>
<p>Leon et al. (2012) Application and evaluation of automated methods to extract neuroanatomical connectivity statements from free text. <span><em>Bioinformatics, 28</em></span>(22):2963-70. doi: 10.1093/bioinformatics/bts542</p>
<p>Acknowledged in:</p>
<p>Zoubarev et al. (2012) Gemma: a resource for the reuse, sharing and meta-analysis of expression profiling data. <span><em>Bioinformatics, 28</em></span>(17):2272-3. doi: 10.1093/bioinformatics/bts430</p>
<h1 id="presentations">Presentations</h1>
<p><span> Poster presentation, Intelligent Systems for Molecular Biology (ISMB) 2016 Conference July 9 &amp; 10, 2016<br />
</span> <span> Poster presentation, BTP/IOP/GSAT 2016 Retreat March 11, 2016<br />
</span></p>
<h1 id="employment-history">Employment history</h1>
<p><span> <span><strong>BCCRC <span class="math inline">|</span> The University of British Columbia</strong></span> Jan.2014 – Aug.2014<br />
</span> <span> <span><em>Research Assistant at the Shah Lab</em></span></span></p>
<ul>
<li><p>Wrote Bash, Python, and R scripts for downstream analysis and pipeline development</p></li>
<li><p>Produced figures and limited text suitable for scientific manuscripts</p></li>
<li><p>Leveraged parallel computing for analyses on a compute cluster to increase efficiency and decrease runtime</p></li>
</ul>
<p><span> <span><strong>BCCRC <span class="math inline">|</span> Provincial Health Services Authority</strong></span> Sept.2013 – Dec.2013<br />
</span> <span> <span><em>Research Intern at the Shah Lab</em></span></span></p>
<ul>
<li><p>Ran automated analysis pipelines for data processing and data generation</p></li>
<li><p>Wrote Bash and Python scripts to rename files, parse data, and wrap third party software</p></li>
<li><p>Documented observations and comments on platforms such as the Atlassian Confluence Wiki, Atlassian Jira issue tracker, and Atlassian Stash Git management system</p></li>
</ul>
<p><span> <span><strong>NSERC Undergraduate Student Research Award (USRA)</strong></span> May.2013 – Aug.2013<br />
</span> <span> <span><em>Undergraduate Summer Research Student in the UBC Department of Statistics</em></span></span></p>
<ul>
<li><p>Contributed to the development of an R toolkit for the management and delivery of data analytic course and research material</p></li>
<li><p>Organized, within a budget, a small meet-and-greet coffee social to acquaint oneself with the UBC Statistics NSERC-USRA students and their supervisors</p></li>
</ul>
<p><span> <span><strong>UBC Centre for High-throughput Biology</strong></span> May.2012 – Aug.2012<br />
</span> <span> <span><em>Linux System Administrator Assistant</em></span></span></p>
<ul>
<li><p>Created and updated documentation and protocols for internal use and for end users</p></li>
<li><p>Created and setup user accounts on desktops, servers, and wikis</p></li>
<li><p>Used a variety of softwares – including Nagios, Bacula, and CFengine – to monitor, backup, and perform system administrative jobs</p></li>
<li><p>Provided troubleshooting services for users and kept record of the issues, challenges, solutions in Request Tracker</p></li>
</ul>
<p><span> <span><strong>UBC Centre for High-throughput Biology</strong></span> Sept.2011 – Apr.2012<br />
</span> <span> <span><em>Bioinformatics Research Assistant at the Pavlidis Lab</em></span></span></p>
<ul>
<li><p>Independently processed and checked the quality of microarray data for the FishManOmics project headed by the Department of Fisheries and Oceans Canada</p></li>
<li><p>Paid attention to detail to analyze, annotate, and curate Gene Expression Omnibus (GEO) datasets that were to be imported into the Gemma database</p></li>
<li><p>Reported issues based on continuous evaluation of lab software (e.g. Gemma) across multiple operating systems including Linux, Windows, and Mac OS</p></li>
</ul>
<h1 id="volunteer-experience">Volunteer experience</h1>
<p><span> <span><em>Helper</em></span>, <span><strong>Software Carpentry workshop</strong></span> Oct.2016<br />
</span> <span> <span><em>Wayfinder</em></span>, <span><strong>St. Paul’s Hospital</strong></span> Feb.2013 – Apr.2013<br />
</span> <span> <span><em>Healthy Heart Resource Centre Clerk</em></span>, <span><strong>St. Paul’s Hospital</strong></span> May.2013 – Aug.2013<br />
</span></p>
</body>
</html>
